src/rules/color-no-invalid-hex/__tests__/index.js

```diff
@@ -0,0 +1,18 @@
+import { ruleTester } from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule(null, tr => {
+  tr.ok("a { color: pink; }")
+  tr.ok("a { color: #000; }")
+  tr.ok("a { something: #000, #fff, #ababab; }")
```

Comment:
You mean use a regex instead of relying on the `color` module? 

Yeah, maybe we should, because it looks like `color` does not handle 4 and 8 digits hexes. We could file PR, instead ...